src/server_manager/web_app/ui_components/outline-share-dialog.ts

```diff
@@ -111,33 +118,31 @@ Polymer({
           inner-h-t-m-l="[[localize('share-description', 'openLink', '<a href=https://securityplanner.org/#/all-recommendations>', 'closeLink', '</a>')]]"
         ></p>
       </div>
-      <div contenteditable="" id="selectableText" style="-webkit-text-size-adjust: 100%;">
-        <p>[[localize('share-invite')]]</p>
+      <div
+        contenteditable=""
+        id="selectableInviteText"
+        style="-webkit-text-size-adjust: 100%;"
+        inner-h-t-m-l="[[localize('share-invite-html')]]"
+      ></div>
+      <div id="copyInvitationIndicator" hidden="">[[localize('share-invite-copied')]]</div>
+      <div class="button-row">
+        <paper-button id="copyButton" on-tap="copyInvite"
+          >[[localize('share-invite-copy')]]</paper-button
+        >
+      </div>
 
-        <p><a href$="{{s3Url}}">{{s3Url}}</a></p>
-        <p>-----</p>
-        <p>[[localize('share-invite-trouble')]]</p>
-        <ol>
-          <li>
-            [[localize('share-invite-copy-access-key')]]
-            <a href="{{accessKey}}">{{accessKey}}</a>
-          </li>
-          <li>
-            [[localize('share-invite-instructions')]]
-            <a
-              href="https://github.com/Jigsaw-Code/outline-client/blob/master/docs/invitation_instructions.md"
-              >https://github.com/Jigsaw-Code/outline-client/blob/master/docs/invitation_instructions.md</a
-            >
-          </li>
-        </ol>
+      <div contenteditable="" id="selectableAccessKey" style="-webkit-text-size-adjust: 100%;">
+        <pre><a href="[[accessKey]]">{{accessKey}}</a></pre>
```

Comment:
No need to use two-way binding here.
```suggestion
        <pre><a href="[[accessKey]]">[[accessKey]]</a></pre>
```